app/presenters/contact_gone_presenter.rb

```diff
@@ -15,6 +17,12 @@ def present
       routes: [
         { path: contact.link, type: "exact" }
       ],
+      content_id: content_id,
     }
   end
+
+  private
```

Comment:
Could you outdent `private`, following the [style guide](https://github.com/alphagov/styleguides/blob/master/ruby.md#classes)?